cc_unitsearch.py

```diff
@@ -0,0 +1,59 @@
+import sys
+import json
+
+from PyQt5.QtWidgets import QApplication, QWidget, QLineEdit, QTableView, QHeaderView, QVBoxLayout
+from PyQt5.QtCore import Qt, QSortFilterProxyModel
+from PyQt5.QtGui import QStandardItemModel, QStandardItem
+
+def pullFromJson():
+    with open('CC_UNITS.json') as json_file:
+        data = json.load(json_file)
+        
+        return data
+
+class CCSearch(QWidget):
+    def __init__(self):
+        super().__init__()
+
+        self.resize(1280, 720)
+        mainLayout = QVBoxLayout()
+        units = pullFromJson()
+
+        model = QStandardItemModel(len(units), 3)
+        model.setHorizontalHeaderLabels(['CC Unit', 'Attila Unit', 'Tier & Type'])
+        for row, unit in enumerate(units):
+            ccUnit = QStandardItem(unit['CC Name'])
+            attilaUnit = QStandardItem(unit['Atilla Name'])
+            unitType = QStandardItem(unit['Unit Type'])
+            model.setItem(row, 0, ccUnit)
+            model.setItem(row, 1, attilaUnit)
+            model.setItem(row, 2, unitType)
+
+        filter_proxy_model = QSortFilterProxyModel()
```

Comment:
What does this do?

Reply:
Since I see this variable is also used by `search_field`, maybe there should be a separate function for this used _only_ by the `createSearchField` or something function

Reply:
After seeing the table, I guess you could make a function like `getFilterProxyModel` which returns the model if it exists or creates it if not and store the value in a class variable